vendor/parser-extended/parser.mly

```diff
@@ -2230,14 +2230,17 @@ expr:
         mkexp_attrs ~loc:$sloc desc attrs }
   | mkexp(expr_)
       { $1 }
-  | let_bindings(ext) IN seq_expr
-      { expr_of_let_bindings ~loc:$sloc $1 $3 }
-  | pbop_op = mkrhs(LETOP) bindings = letop_bindings IN body = seq_expr
+  | let_bindings(ext) mkrhs(IN) seq_expr
+      { expr_of_let_bindings ~loc:$sloc ~loc_in:$2.loc $1 $3 }
+  | pbop_op = mkrhs(LETOP) bindings = letop_bindings _in_kw=IN body = seq_expr
```

Comment:
Why is it prefixed with `_` ?

Reply:
Because it is unused : we only access the loc, and the way menhir does its cuisine, asking for the loc does make the variable used.

Reply:
Good to know. Seems like a bug in menhir.